modules/ebi-metagenomics/fraggenescanrs/main.nf

```diff
@@ -0,0 +1,59 @@
+process FRAGGENESCANRS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/frag_gene_scan_rs:1.1.0--h4349ce8_0':
+        'biocontainers/frag_gene_scan_rs:1.1.0--h4349ce8_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    val(training_file_name)
+
+    output:
+    tuple val(meta), path("*.ffn.gz"), emit: nucleotide_fasta
+    tuple val(meta), path("*.faa.gz"), emit: amino_acid_fasta
+    tuple val(meta), path("*.out.gz"), emit: gene_annotations
+    tuple val(meta), path("*.gff.gz"), emit: gff
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def catcommand = fasta.name.endsWith(".gz") ? "zcat" : "cat"
+    """
+    ${catcommand} ${fasta} | FragGeneScanRs \\
+        $args \\
```

Comment:
We need to remember about args in pipelines. Example, FGS had those https://github.com/EBI-Metagenomics/nf-modules/blob/main/modules/ebi-metagenomics/fraggenescan/main.nf#L29

command I tested:
`FragGeneScanRs -s genomic.fna -t illumina_10 -w 0 -o fgsR -p 4`
`-w` should be in args I think

Reply:
Yes, the complete one is 0 by default:

```
-w, --complete <complete> The input sequence has complete genomic sequences; not short sequence reads. [default: 0]
```
The train "mode" is a mandatory arg -- I think having it there is better than in a config file as it's a mantdatory arg in this version of FGS.